### PR TITLE
Apt-get cleanup

### PR DESCRIPTION
**Pull request by vkumra-broad**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by vkumra-broad** [comment]
jenkins retest

**Comment by vkumra-broad** [comment]
jenkins retest

**Comment by vkumra-broad** [comment]
jenkins retest

**Comment by vkumra-broad** [comment]
jenkins retest

File: jupyter-docker/Dockerfile [hunk 1]
```diff
@@ -223,7 +229,9 @@ RUN R -e 'install.packages(c( \
  && R -e 'devtools::install_github("apache/spark@v2.0.2", subdir="R/pkg")' \
  && mkdir -p /home/jupyter-user/.rpackages \
  && echo "R_LIBS=/home/jupyter-user/.rpackages" > /home/jupyter-user/.Renviron \
- && chown -R $USER:users /home/jupyter-user/.rpackages
+ && chown -R $USER:users /home/jupyter-user/.rpackages \
+ && apt-get clean \
```
Comment (rtitle) [inline jupyter-docker/Dockerfile]: I think this one should be in the previous layer? The previous one does the apt-gets, this one runs the `R install`s.

**Review by rtitle — CHANGES_REQUESTED** [review]
1 of them is in the wrong place - otherwise looks good, shaving off ~100MB is definitely valuable.

File: jupyter-docker/Dockerfile [hunk 2]
```diff
@@ -45,7 +45,9 @@ RUN echo "deb $DEBIAN_REPO/debian jessie main"                   > /etc/apt/sour
  && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - \
  && apt-get update \
  && apt-get install -yq --no-install-recommends \
-    google-cloud-sdk
+    google-cloud-sdk \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
```
Comment (kyuksel) [inline jupyter-docker/Dockerfile]: I wonder if it makes sense to combine `apt-get clean && rm -rf /var/lib/apt/lists/*` into an alias like so:

```alias apt-get-cleanup='apt-get clean && rm -rf /var/lib/apt/lists/*'```

**Comment by rtitle** [comment]
Could you post some size results for before and after this change? I checked out this branch and built the docker file and strangely the image size increased:
```
rtitle@~$ docker images
REPOSITORY                                          TAG                 IMAGE ID            CREATED             SIZE
rt-apt-get-cleanup                                  latest              bc4e1fc765fb        29 minutes ago      2.61GB
```

This doesn't seem right so I'm wondering if I did something wrong..

**Comment by vkumra-broad** [comment]
```
<none>                              <none>                    ce6758b189dc        46 seconds ago      2.72GB 
<none>                              <none>                    ec209351a67b        19 minutes ago      2.84GB
```

First one is with apt-get cleanup, second one is without it

**Comment by vkumra-broad** [comment]
jenkins retest

**Review by rtitle — APPROVED** [review]

**Pull request merged**